src/Compilers/Core/Portable/DiagnosticAnalyzer/SuppressMessageAttributeState.cs

```diff
@@ -34,6 +34,10 @@ private static bool TryGetTargetScope(SuppressMessageInfo info, out TargetScope
         private readonly Compilation _compilation;
         private GlobalSuppressions? _lazyGlobalSuppressions;
         private readonly ConcurrentDictionary<ISymbol, ImmutableDictionary<string, SuppressMessageInfo>> _localSuppressionsBySymbol;
+
+        // These are StrongBoxes because 'null' is a valid symbol value to compute for these, and as such, we can't use
+        // the null value to indicate 'not yet computed'.
```

Comment:
feel free to delete this comment as well.  i just thought it might be helpful for someone asking why a reference type needed to be wrapped in a strongbox.